docs/source/tutorial/set_up_data.md

```diff
@@ -1,94 +1,48 @@
+The spaceflights tutorial makes use of three fictional datasets of companies shuttling customers to the Moon and back:
 
-The spaceflight tutorial has three files and uses two data formats: `.csv` and `.xlsx`. Download and save the files to the `data/01_raw/` folder of your project directory:
+* `companies.csv` contains data about space travel companies, such as their location, fleet count and rating
+* `reviews.csv` is a set of reviews from customers for categories, such as comfort and price
+* `shuttles.xlsx` is a set of attributes for spacecraft across the fleet, such as their engine type and passenger capacity
 
-* [reviews.csv](https://kedro-org.github.io/kedro/reviews.csv)
-* [companies.csv](https://kedro-org.github.io/kedro/companies.csv)
-* [shuttles.xlsx](https://kedro-org.github.io/kedro/shuttles.xlsx)
-
-Here are some examples of how you can [download the files from GitHub](https://www.quora.com/How-do-I-download-something-from-GitHub) to the `data/01_raw` directory inside your project:
-
-Using [cURL in a Unix terminal](https://curl.se/download.html):
-
-<details>
-<summary><b>Click to expand</b></summary>
+You will use the data to train a model to predict the price of shuttle hire. However, before you get to train the model, you will need to prepare the data for model building by combining the files to create a model input table.
 
-```bash
-# reviews
-curl -o data/01_raw/reviews.csv https://kedro-org.github.io/kedro/reviews.csv
-# companies
-curl -o data/01_raw/companies.csv https://kedro-org.github.io/kedro/companies.csv
-# shuttles
-curl -o data/01_raw/shuttles.xlsx https://kedro-org.github.io/kedro/shuttles.xlsx
-```
-</details>
-
-Using [cURL for Windows](https://curl.se/windows/):
-
-<details>
-<summary><b>Click to expand</b></summary>
-
-```bat
-curl -o data\01_raw\reviews.csv https://kedro-org.github.io/kedro/reviews.csv
-curl -o data\01_raw\companies.csv https://kedro-org.github.io/kedro/companies.csv
-curl -o data\01_raw\shuttles.xlsx https://kedro-org.github.io/kedro/shuttles.xlsx
-```
-</details>
+The data comes in two different formats: `.csv` and `.xlsx`. Kedro supports a number of different data types, and those supported can be found in the API documentation.
 
-Using [Wget in a Unix terminal](https://www.gnu.org/software/wget/):
+> If you are using the tutorial created by the spaceflights starter, you can omit this step.
 
-<details>
-<summary><b>Click to expand</b></summary>
+Download and save the files to the `data/01_raw` folder of your project:
 
-```bash
-# reviews
-wget -O data/01_raw/reviews.csv https://kedro-org.github.io/kedro/reviews.csv
-# companies
-wget -O data/01_raw/companies.csv https://kedro-org.github.io/kedro/companies.csv
-# shuttles
-wget -O data/01_raw/shuttles.xlsx https://kedro-org.github.io/kedro/shuttles.xlsx
```

Comment:
This is removed, does that mean users need to download these files one by one and put them in the `data` folder?

Reply:
Yes, and I think that's OK. The amount of complexity/space devoted to providing options for using the terminal compared to just clicking on each of the links is unnecessary in my view. And I'm hoping most people will use the starter anyway.